Sources/CGColorSpace+Convenience.swift

```diff
@@ -0,0 +1,46 @@
+import CoreGraphics
+
+
+/// Some useful extensions for convenience access to the most common color spaces needed when working with Core Image.
+public extension CGColorSpace {
+
+    /// The standard Red Green Blue (sRGB) color space.
+    static var sRGBColorSpace: CGColorSpace? { CGColorSpace(name: CGColorSpace.sRGB) }
```

Comment:
Because there already exist static properties with those names, but they only refer to the name of the color space, not the color spaces themself (like you see here in the implementation of the property).